java-checks/src/test/files/checks/security/IntegerToHexStringCheck.java

```diff
@@ -0,0 +1,18 @@
+class A {
```

Comment:
So I understand that human readable implementations can be made:

- by using a StringBuilder / StringBuffer
- by outputting on console output - in this case, why not add the slf4j.Logger interface ? isn't it the equivalent idea?
- should check for [StringJoiner](https://docs.oracle.com/javase/8/docs/api/java/util/StringJoiner.html) as well?

Reply:
* Using slf4j.Logger I don't see how you append bytes converted to hex. I think this will be done with a StringBuilder before calling the log method.
* I'm going to add StringJoiner#add